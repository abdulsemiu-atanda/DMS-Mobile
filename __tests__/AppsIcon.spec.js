import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'

import AppsIcon from '../components/icons/AppsIcon.react'

describe('AppsIcon', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<AppsIcon tintColor='blue' />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
