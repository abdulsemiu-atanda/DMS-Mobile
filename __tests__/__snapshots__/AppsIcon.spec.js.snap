// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppsIcon renders correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={false}
  ellipsizeMode="tail"
  style={
    Array [
      Object {
        "color": "blue",
        "fontSize": 32,
      },
      undefined,
      Object {
        "fontFamily": "Ionicons",
        "fontStyle": "normal",
        "fontWeight": "normal",
      },
    ]
  }
>
  ï„Š
</Text>
`;
